openstad/openstad-management-panel
# Feature/copy users and rights

# Description
On the Admin panel, when you either create, copy or import sites, you will have the option to import users from a selected site to the new site. This will happen after the site has been created.

## Issue reference

Fixes # (issue)
https://trello.com/c/W4LNEp4J

## Dependencies
https://github.com/openstad/openstad-oauth2-server/pull/115 is necessary for this pr
## Type of change
feature

## Tests
For the copy site functionality
- Log in to admin panel
- Create a website with the create site functionality with a few users and roles of your choosing (and log in with the users on that specific site)
- Use the copy site function to create a website based of the previously created site
- Check the checkbox that enables the copying of users and rights

The create and import pages work in the same manner with the exception that you will need to check the checkbox and search for a specific site in the combobox.
